base/applications/network/ftp/main.c

```diff
@@ -227,7 +227,10 @@ int main(int argc, const char *argv[])
 		(void) strcpy(home, pw->pw_dir);
 	}
 #endif
-        strcpy(home, "C:/");
+	if (GetEnvironmentVariableA("SystemDrive", home, ARRAYSIZE(home)))
+		strcat(home, "/");
+	else
+		strcpy(home, "C:/");
```

Comment:
Maybe worth logging a DPRINT1 with GetLastError here as GetEnvironmentVariableA not supposed to fail for SystemDrive

Reply:
We should avoid using macros that resolve to ntdll exports in these sort of utilities.